### PR TITLE
🪟 🐛 Add space between job is in progress and schema changes detected indicator in connections table

### LINKED ISSUE #22629: [Bug] Missing space between job is in progress and schema changes detected indicator in connections table
Image

### PR DESCRIPTION
**Pull request by dizel852**
## What
Resolves https://github.com/airbytehq/airbyte/issues/22629

## How
Add `gap` for `flex` `<StatusCell />` container

Before:
Screenshot at Feb 09 20-29-58

After fix:
Screenshot at Feb 09 20-31-02

### CODE REVIEW

**Comment by edmundito** [comment]
@dizel852 Looking a lot better but I was wondering if one more thing can be done. When pressing the "sync now" button next to a connection with a breaking/non breaking change indicator will lead the whole table to resize because the column width has changed.

Is it possible to assign a fixed width to that column that can fit all the states so that the table doesn't resize when there's a state change?

**Comment by dizel852** [comment]
> @dizel852 Looking a lot better but I was wondering if one more thing can be done. When pressing the "sync now" button next to a connection with a breaking/non breaking change indicator will lead the whole table to resize because the column width has changed.
> 
> Is it possible to assign a fixed width to that column that can fit all the states so that the table doesn't resize when there's a state change?

Sure! Will update the fix

File: airbyte-webapp/src/components/EntityTable/components/StatusCell.module.scss [hunk 1]
```diff
@@ -1,7 +1,9 @@
+@use "scss/variables";
```
Comment (krishnaglick) [inline airbyte-webapp/src/components/EntityTable/components/StatusCell.module.scss:1]: This can be removed.

**Review by krishnaglick — APPROVED** [review]

File: airbyte-webapp/src/components/EntityTable/components/StatusCell.module.scss (hunk 1 above)
Comment (dizel852) [reply airbyte-webapp/src/components/EntityTable/components/StatusCell.module.scss:1]: Oh, yes, missed that. Thanks!

**Pull request merged**